Tasks/AzureCLIV1/azureclitask.ts

```diff
@@ -142,9 +142,7 @@ export class azureclitask {
 
     private static throwIfError(resultOfToolExecution: IExecSyncResult): void {
         if (resultOfToolExecution.code != 0) {
-            if (resultOfToolExecution.stderr) {
-                throw resultOfToolExecution;
-            }
+            throw resultOfToolExecution;
```

Comment:
Should you put some generic error message like "Login failed with a non-zero exit code" in case there is no stderr

Reply:
done